src/components/Counter.js

```diff
@@ -6,34 +6,42 @@ class Counter extends Component {
     incrementIfOdd = () => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+
+        if(this.props.count%2!==0){
+            return(
+                this.props.increment()
+            )
```

Comment:
All you'll need to do here is to invoke the `increment` function for running the action creator, so there's no need to return its result.